StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/StyleCop.Analyzers.nuspec

```diff
@@ -19,8 +19,10 @@
 
     <!-- Binaries and symbols -->
     <file src="bin\$Configuration$\StyleCop.Analyzers.dll" target="analyzers\dotnet\cs" />
+    <file src="bin\$Configuration$\**\StyleCop.Analyzers.resources.dll" target="analyzers\dotnet\cs" />
```

Comment:
:memo: Need to make sure the satellite assemblies are included in the distribution. :smile: